src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1201,15 +1279,16 @@ void S6Exporter::ExportEntityPeep(RCT2SpritePeep* dst, const Peep* src)
     }
     if (generateName)
     {
-        if (src->Is<Staff>())
+        auto* staff = src->As<Staff>();
+        if (staff != nullptr)
```

Comment:
```suggestion
        if (auto* staff = src->As<Staff>(); staff != nullptr)
```

Reply:
I have to disagree - the old code is much easier to read.